README.md

```diff
@@ -4,7 +4,19 @@ Terraform module for [Azure Database for PostgreSQL - Flexible Server](https://d
 ## Example
 
 ```hcl
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+  alias                      = "postgres_network"
+  subscription_id            = var.aks_subscription_id
+}
+
 module "postgresql" {
+
+  providers = {
+    azurerm.postgres_network = azurerm.postgres_network
+  }
+  
```

Comment:
should only be needed if you are renaming the provider I think
```suggestion
```

Reply:
Is this not needed on teams module reference to init with correct provider?

Reply:
i don't think so, all providers should be passed through, you can test it from the example folder

Reply:
Seems it is needed
```
 The child module requires an additional configuration for provider
│ hashicorp/azurerm, with the local name "azurerm.postgres_network".
```
GH Action: https://github.com/hmcts/terraform-module-postgresql-flexible/actions/runs/4562095063/jobs/8048854878?pr=31

PR: https://github.com/hmcts/terraform-module-postgresql-flexible/pull/31/commits/4497cee6f5735f1ab26a2808f587468be18f61df